tasks/css.js

```diff
@@ -106,6 +106,8 @@ gulp.task('css:lint', () => gulp.src(`${pathBuilder.cssDistDir}/**/*.css`)
  */
 gulp.task('css:bundle', () => {
 
+    const { applyRevision } = config;
```

Comment:
Looking through the rest of the file I think you should just call `config.applyRevision` below, keeping the code style consistent throughout the project.

Reply:
@DamianMullins Should I change this [here](https://github.com/justeat/gulp-build-fozzie/blob/eacb94812bf7d30bf24b6904340c1de35429115d/tasks/javascript.js#L166) too?

Reply:
Yeah, change in both places.